README.md

```diff
@@ -159,6 +161,17 @@ bohb_tune_search = TuneSearchCV(SGDClassifier(),
 
 bohb_tune_search.fit(X_train, y_train)
 
+# Define the `param_dists using the SearchSpace API
+# This allows the specification of sampling from discrete and 
+# categorical distributions (below for the `learning_rate` scheduler parameter)
+param_dists = {
+    'loss': tune.choice(['squared_hinge', 'hinge']),
+    'alpha': tune.loguniform(1e-4, 1e-1),
+    'epsilon': tune.uniform(1e-2, 1e-1),
+    'learning_rate': tune.sample_from(lambda _: np.random.choice(['constant', 'optimal'], p=[0.2, 0.8])),
```

Comment:
hmm does the `sample_from` actually work with `hyperopt`?

Reply:
Indeed it doesn't! I just pushed a change removing it. I must have polluted my notebook when I was testing this out.